lib/perl/Genome/InstrumentData/AlignmentResult.pm

```diff
@@ -1726,9 +1726,15 @@ sub get_merged_bam_to_revivify_per_lane_bam {
 sub get_merged_alignment_results {
     my $self = shift;
     # Always load from the database, since other merged results may have committed since we updated the UR cache
+    my %params;
+    for my $param ($self->__meta__->params) {
+        my $name = $param->property_name;
+        $params{$name} = $self->$name;
+    }
```

Comment:
@brummett, can't you go from a UR::Object to a UR::BoolExpr?  I thought there was something like,

```
my %params = UR::BoolExpr->resolve($self)->params_list;
```